grafana-formula/grafana-formula.spec

```diff
@@ -19,7 +19,7 @@
 %define fname grafana
 %define fdir  %{_datadir}/susemanager/formulas
 Name:           grafana-formula
-Version:        0.2.2
+Version:        0.2.3
```

Comment:
Since the master branch has `0.4.x` this is safe, as this will be able to upgrade to SUSE Manager 4.1 or 4.1.